include/tscore/ink_platform.h

```diff
@@ -60,7 +60,7 @@
 struct ifafilt;
 #include <net/if.h>
 
-#ifdef HAVE_STDLIB_H
+#if __has_include(<stdlib.h>)
```

Comment:
When would we ever not have stdlib.h ... it is a standard C header
We should remove the check for this

Reply:
I agree.

Reply:
Yeah, I agree with @dragon512 on this one . I can't think of any system that doesn't need stdlib.h. But at the same time, this is to remove an unnecessary check in CMakeLists.txt. If there is any doubt I'm ok with this change.